### DISCUSSION TITLE
Problems with multiple video in the same space

### DISCUSSION THREAD

**Discussion by Dllamasr** [mozilla/hubs#4082] [Spoke Questions] [answered]
Hello! 
I've created a very basic space to compilate all the songs for Eurovision Song Contest this year.
https://hubs.mozilla.com/KtEppww/eurovision-song-contest-2021/

I have an issue, because when I enter into the space, it seems that all the songs starts autoplaying altogether for just half a second (though I did not check that box in Spoke) making an annoying noise and then they stop (as they are supposed to). Also, when I try to play any of them, sometimes they get slow or suddenly stops... I suppose it's because of the 40 videos embedded at once... How do you think I can solve this issue? Maybe clearing the songs from Spoke to enlighten the model and pinning them directly from Mozilla afterwards?

 Thank you!

**Comment by johnshaughnessy** [comment, marked as answer]
>  when I enter into the space, it seems that all the songs starts autoplaying altogether for just half a second

This is a known issue. The videos load and starts playing before the paused state or correct volume level is applied. I agree it is annoying.

> when I try to play any of them, sometimes they get slow or suddenly stops... I suppose it's because of the 40 videos embedded at once... How do you think I can solve this issue? 

Currently we don't have "interest management" in the client that would allow a performant experience with 40 videos in the room. Instead, the client will simply try to play all 40 videos simultaneously. I don't know of any easy fix to this problem besides making 40 different spaces or loading 1-2 videos at a time while you're in a room.
